CONTRIBUTING.md

```diff
@@ -4,8 +4,9 @@
 
 We're using [pnpm workspaces](https://pnpm.js.org/en/workspaces).
 
+1. `corepack enable && corepack install`
```

Comment:
Corepack is experimental still, so not sure I want to commit to it in this project